gramps/plugins/graph/gvhourglass.py

```diff
@@ -444,3 +455,8 @@ def add_menu_options(self, menu):
         ahnentafelnumvisible.set_help(
             _("Show Sosa / Sosa-Stradonitz / Ahnentafel number under all others informations."))
         menu.add_option(category_name, "ahnentafelnum", ahnentafelnumvisible)
+
+        datesymbol = BooleanOption(_("Add Symbol for date"), False) # 10826
+        datesymbol.set_help(
+            _("Add a symbol for date. o for birth, + for decease and x for wedding."))
```

Comment:
Suggested corrections: 

* Add a symbol for date.  >  Show symbol for events.

* decease > decease**d**

* wedding  > marriage

Reply:
nice idea, 
English is not my best language :)